backends/k8s/client.go

```diff
@@ -183,46 +184,49 @@ func (c *Client) WatchPrefix(prefix string, waitIndex uint64, stopChan chan bool
 			ver := tprs.Metadata.ResourceVersion
 			return convertResourceVersionToUint(ver, prefix)
 		default:
-			return 0, nil
+			// We aren't tracking this key, default to 10 second refresh.
+			time.Sleep(10 * time.Second)
```

Comment:
If we're not tracking this key does it make sense to do it even less frequently than 10s?